test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java

```diff
@@ -674,6 +681,12 @@ void canSeeDaycount() {
       assertContinueState();
 
       assertThat(output, is("Returned: 7302\n"));
+
+      /*
+       ApiRequest.parseStatus sets a password hash which persists to other tests and causes them to pass or fail
+       based upon whether this test was run first, or not.  Explicitly clear the hash when this test ends.
+      */
+      ApiRequest.setPasswordHash("");
```

Comment:
put this in a `finally` block so it runs even if the test fails.

Reply:
OK.  Update coming.